### DISCUSSION TITLE
Help needed configuring editor command

### DISCUSSION THREAD

**Discussion by uwla** [qutebrowser/qutebrowser#6352] [Q&A] [answered]
Hello! The default value for `editor.command` is `["gvim", "-f", "{file}", "-c", "normal {line}G{column0}l"]`. I changed `gvim` to `vim` since I'm using vim, but it is not working. When I press Ctrl-E it does nothing. It does not open vim editor. Can someone please help me?

**Comment by uwla** [comment, marked as answer]
Never mind... I should have realized that vim can only start from a terminal or tty. So the correct way to do is

```python
c.editor.command = [TERM, '-e', 'vim {file}']
```

Where `TERM` is the user's terminal.

**Reply by The-Compiler** [reply]
The options to use depend on the terminal, though. Some require passing the thing to run as multiple arguments (with `-e` consuming all arguments after it). Some use `-x` instead of `-e` (and perhaps other flags as well). Some additionally require something like `--no-fork` so they don't exit immediately.
